nucypher/crypto/powers.py

```diff
@@ -211,5 +211,12 @@ def generate_kfrags(self, bob_pubkey_enc, signer, label, m, n) -> Union[UmbralPu
         # TODO: salt?  #265
 
         __private_key = self.umbral_keying_material.derive_privkey_by_label(label)
-        kfrags = pre.split_rekey(__private_key, signer, bob_pubkey_enc, m, n)
+        kfrags = pre.generate_kfrags(delegating_privkey=__private_key,
```

Comment:
OK, sure.

So now we get back (on each and every kfrag) a 'mode' portion, which is a constant_sorrow constant, determined by the combination of which keys to sign (here, for the moment neither).

Got it.